Resources/Prototypes/_RMC14/Entities/Objects/Medical/hypospray.yml

```diff
@@ -21,6 +21,8 @@
     sprite: _RMC14/Objects/Medical/hypospray.rsi
   - type: Hypospray
     transferAmount: 5
+    onlyAffectsMobs: false
+    injectOnly: true
```

Comment:
`    onlyAffectsMobs: false
    injectOnly: true`
won't this make it so you can't refill the hypospray at all, while still allowing it to inject into containers?

the hypospray currently can't 'draw' from a container, but can be filled by a container, since it doesn't have the 'InjectorToggleMode', which is what allows something like syringes, droppers and such, to switch between draw and inject. 

if you want it to just stop being able to inject into containers then remove line 25 'injectoonly: true' and change line 24 'onlyaffectsmobs: false' to 'onlyaffectmobs: true'

i could very well be wrong, since i  am still very new to coding, but that is how i had understod it

Reply:
Injecting into containers is fine. It's the fact you can draw from them that isn't.